dev/com.ibm.ws.concurrent.persistent/resources/com/ibm/ws/concurrent/persistent/resources/CWWKCMessages.nlsprops

```diff
@@ -55,9 +55,9 @@ CWWKC1520.out.of.range=CWWKC1520E: Configured value {0} for {1} is not within th
 CWWKC1520.out.of.range.explanation=A value that is outside of the allowed range is configured for the specified property.
 CWWKC1520.out.of.range.useraction=Configure a value within the allowed range.
 
-CWWKC1521.less.than.min=CWWKC1521E: Configured value {0} for {1} cannot be less than the configured value for {2}, which is {3}.
-CWWKC1521.less.than.min.explanation=Configuring a larger retryInterval encourages other servers to retry the failed task instead of the server on which it failed.
-CWWKC1521.less.than.min.useraction=Configure the retryInterval property so that it is larger than the missedTaskThreshold.
+CWWKC1521.not.compatible=CWWKC1521E: Configuration attribute {0} is not valid when {1} is enabled.
```

Comment:
**change**
Configuration attribute {0} is not valid when {1} is enabled
**to**
The {0} configuration attribute is not valid when the {1} configuration attribute is enabled.